cypress/support/commands.js

```diff
@@ -183,98 +183,27 @@ Cypress.Commands.add('selectSubjectLikeToTeach', (subject) => {
 	cy.clickOnContinueButton();
 });
 
-Cypress.Commands.add('youLiveIn', (location) => {
-	if (location == 'UK') {
-		cy.get('#teacher-training-adviser-steps-uk-or-overseas-uk-or-overseas-uk-field').click();
-	} else {
-		cy.get(
-			'#teacher-training-adviser-steps-uk-or-overseas-uk-or-overseas-overseas-field'
-		).click();
-		cy.clickOnContinueButton();
-		cy.get('#teacher-training-adviser-steps-overseas-country-country-id-field').select(
-			location
-		);
-	}
-	cy.clickOnContinueButton();
-});
-
-/*Cypress.Commands.add("whereDoYouLive", (location) => {
-	if (location == "UK") {
-		cy.get("#uk-or-overseas-uk-or-overseas-uk-field").click();
-	} else {
-		cy.get("#uk-or-overseas-uk-or-overseas-overseas-field").click();
-		cy.get(".govuk-button").click();
-		cy.get("#overseas-country-country-id-field").select(location);
-	}
-	cy.get(".govuk-button").click();
-});*/
-
-Cypress.Commands.add('whereDoYouLive', (location, studying, equivalent) => {
-	switch (true) {
-		case studying:
-			if (location == 'UK') {
-				cy.get(
-					'#teacher-training-adviser-steps-uk-or-overseas-uk-or-overseas-uk-field'
-				).click();
-			} else {
-				cy.get(
-					'#teacher-training-adviser-steps-uk-or-overseas-uk-or-overseas-overseas-field'
-				).click();
-				cy.clickOnContinueButton();
-				cy.get('#teacher-training-adviser-steps-overseas-country-country-id-field').select(
-					location
-				);
-			}
-			break;
-		case equivalent:
-			if (location == 'UK') {
-				cy.get(
-					'#teacher-training-adviser-steps-uk-or-overseas-uk-or-overseas-uk-field'
-				).click();
-			} else {
-				cy.get(
-					'#teacher-training-adviser-steps-uk-or-overseas-uk-or-overseas-overseas-field'
-				).click();
-				cy.clickOnContinueButton();
-				cy.get('#teacher-training-adviser-steps-overseas-country-country-id-field').select(
-					location
-				);
-			}
-			break;
-		default:
-			if (location == 'UK') {
-				cy.get(
-					'#teacher-training-adviser-steps-uk-or-overseas-uk-or-overseas-uk-field'
-				).click();
-			} else {
-				cy.get(
-					'#teacher-training-adviser-steps-uk-or-overseas-uk-or-overseas-overseas-field'
-				).click();
-				cy.clickOnContinueButton();
-				cy.get('#teacher-training-adviser-steps-overseas-country-country-id-field').select(
-					location
-				);
-			}
-	}
-
+Cypress.Commands.add("whichCountryDoYouLiveIn", (country) => {
+	cy.contains("Which country do you live in?")
+		.invoke("attr", "for")
+		.then(function (val) {
+			let id = "#" + val;
+			cy.get(id).select(country);
+		});
 	cy.clickOnContinueButton();
 });
 
-Cypress.Commands.add('enterUKTelephoneNumber', (number, equivalent) => {
-	if (equivalent) {
-		cy.get('#teacher-training-adviser-steps-uk-telephone-telephone-field').type('0125234490');
+Cypress.Commands.add("whereDoYouLive", (location) => {
```

Comment:
Could this be changed to a `boolean` and something like:

```
Cypress.Commands.add("doYouLiveInTheUk", uk) => {
  const location = uk ? "UK" : "Overseas";
  cy.contains(location).click();
  cy.clickOnContinueButton();
}
```

Reply:
Command **cy.contains(location).click()**  is not working if location is "UK" ( there are two places where UK is present. First one is UK radio button and another is GOV.UK so this command doesn't clicks on expected one  ). Alternatively I've written code as below. Please let me know if you're not happy.
Cypress.Commands.add("doYouLiveInTheUk", (location)  =>{
	if(location)
	{
		cy.get(".govuk-radios__label").first().click();
	}
	else
	{
		cy.contains("Overseas").click();
	}			
	    cy.clickOnContinueButton();
  });

Reply:
Ah ok, that's fine in that case, can we just change the argument from `location` to `uk` so that the if statement makes more sense though?

Reply:
done